Scheduler/REST/CSharpClient2/Program.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.Serialization;
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+
+/*
+ * This is a sample client for the new REST API of HPC Pack. Get the API spec at:
+ * http://download.microsoft.com/download/B/D/B/BDB8782A-FAAF-457D-AF3D-0B157FEEDF4C/New%20Set%20of%20HPC%20Pack%20Scheduler%20REST%20API.pdf
+ */
+
+namespace RestClient2

```

Comment:
Is it possible that we give this class/sample a better name to inform its purpose and difference with the old one more clearly? How about something likes `RestClient2016` (to indicate the set of corresponding REST API is new to HPC Pack 2016) and describe it in the comment above?

Reply:
OK. See my new commit for the change.